index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+        integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+    <!-- Header Section -->
+    <section class="desktop-nav">
+        <div class="first-navbar">
+            <nav>
+                <ul>
+                    <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+                    <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                    <li><a href="#" class=" fw-bold">English</a></li>
+                    <li><a href="#" class="fw-bold">LogIn</a></li>
+                    <li><a href="#" class="fw-bold">LogOut</a></li>
+
+                </ul>
+            </nav>
+        </div>
+        <div class="second-navbar">
+            <div>
+                <h1 class="display-4 fw-bold m-2">AHMADI GYM</h1>
+            </div>
+            <div>
+            <nav>
+                <ul>
+                    <li><a href="about.html" class="fw-bold
+                        ">About</a></li>
+                    <li><a href="#programs-section" class="fw-bold
+                        ">Program</a></li>
+                    <li><a href="#join-section" class="fw-bold">Join</a></li>
+                    <li><a href="#featured-section" class="fw-bold
+                        "></a></li>
+                    <li class="last-list-item" class="fw-bold
+                    "><a href="#" class="fw-bold">AHMADI GYM</a></li>
+
+                </ul>
+            </nav>
+            </div>
+        </div>
+    </section>
+    <section class="first-page">
+        <header class="header-sect">
+            <a href="#"><i class="fas fa-bars fa-2xl" id="hamburger-menu"></i></a>
+        </header>
+        <article class="intro-sect">
+            <p class="heading-intro fw-bold">Stop Thinking Start Doing!</p>
+            <h2 class="fw-bold">Work-Sweat-Achieve</h2>
+            <h2 class="fw-bold"> Nothing Will Work Unless You Do</h2>
+            <h2 class="fw-bold">2023</h2>
```

Comment:
- [ ] I would kindly suggest you make this tag an `h1` tag as it is advisable to have atleast one h1 tag in a webpage for readability, accessibility and SEO (Search Engine Optimization) sake